phases/ephemeral/docs.md

```diff
@@ -337,8 +337,12 @@ The right to invoke `path_rename` with the file descriptor as the target directo
 The right to invoke `path_filestat_get`.
 
 - <a href="#rights.path_filestat_set_size" name="rights.path_filestat_set_size"></a> `path_filestat_set_size`
-The right to change a file's size (there is no `path_filestat_set_size`).
+The right to change a file's size.
 If `path_open` is set, includes the right to invoke `path_open` with [`oflags::trunc`](#oflags.trunc).
+Note: there is no function named `path_filestat_set_size`. This follows POSIX design,
+which only has `ftruncate` and does not provide `ftruncateat`. While such function
+would definitely be useful, it would be difficult to implement `path_filestat_set_size`
+efficiently on most existing systems.
```

Comment:
Would it be difficult to implement efficiently? It seems we could implement this by just opening the file and doing `ftruncate`, which would be reasonable. My understanding is that the main reason we don't have `path_filestat_set_size` is that we don't currently have any use cases for it, since no code written for POSIX systems would use it.

Reply:
Here _to implement efficiently_ is intended to mean _to implement using one syscall only_, as we consider the cost of other calls negligible.

How about 
> While such function would be desirable from the API design perspective, there are virtually no use cases for it since no code written for POSIX systems would use it. Moreover, implementing it would require multiple syscalls, leading to inferior performance.

Reply:
@sunfishcode bump

Reply:
Thanks for the bump! That wording sounds good to me.

Reply:
I pushed the new wording.